### ISSUE TITLE
Not triggered

### ISSUE THREAD

**Issue by Hueristic** [minimal3dp/minimal3dp.github.io#12]
Fails when should be triggered reads open.

How to test with a meter to see if probe needs replacing before tearing into all the wiring.

**Comment by minimal3dp** [comment]
You should be able to invert the probe trigger via the firmware.

**Comment by Hueristic** [comment]
Correct, my reply was to your "how can we make this tutorial better" comment. Also the octocouplker is burned out so doesn't matter if you invert it or enable the resistor.
